lib/slack.py

```diff
@@ -8,25 +8,42 @@ def __init__(self, **kwargs):
         self._client = None
 
 
-    def send_message(self, message):
+    def send_message(self, message, params= None):
```

Comment:
What you can do is set params to what you have currently as `default_params` rather than `None`, that way you need not check at all if params is defined using an if statement, you can just do the api_call and know that it will always be defined correctly